cdap-docs/vars

```diff
@@ -49,6 +49,9 @@ GIT_BRANCH_CDAP_INGEST="1.3.0"
 GIT_BRANCH_CDAP_APPS="develop"
 GIT_VERSION_CDAP_APPS="0.8.0"
 
+# Used by examples manual
+GIT_BRANCH_CDAP_GUIDES="release/cdap-3.5-compatible" # Using until 3.6 versions available
+
```

Comment:
This should be:
```
GIT_BRANCH_CDAP_GUIDES="develop"
```
and then PR https://github.com/caskdata/cdap/pull/6839 can be closed.